ramls/instance.json

```diff
@@ -27,6 +27,36 @@
       "type": "string",
       "description": "The metadata source and its format of the underlying record to the instance record. (e.g. FOLIO if it's a record created in Inventory;  MARC if it's a MARC record created in MARCcat or EPKB if it's a record coming from eHoldings)"
     },
+    "relatedInstances": {
+      "description": "Array of related instances",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "description": "Id of the related instance join",
+            "type": "string"
+          },
+          "instanceId": {
+            "description": "Id of the instance",
+            "type": "string"
+          },
+          "relatedInstanceId": {
+            "description": "Id of the related instance",
+            "type": "string"
+          },
+          "relatedInstanceTypeId": {
+            "description": "Id of the related instance type",
+            "type": "string"
+          }
+        }, 
+        "additionalProperties": false,
+        "required": [
+          "relatedInstanceId",
+          "relatedInstanceTypeId"
+        ]
```

Comment:
Should instance ID also be required?

As I understand it, both are needed for a related instance to make sense. Since we are exposing that aspect and not using the host instance's ID, I think we need it to be provided here.

Reply:
Yes it should. Thank you.